components/Front/index.js

```diff
@@ -132,6 +134,28 @@ const Front = ({
   finite,
   hasOverviewNav
 }) => {
+  const [isRefetching, setIsRefetching] = useState(false)
+  const now = new Date()
+  const dailyUpdateTime = new Date(
+    now.getFullYear(),
+    now.getMonth(),
+    now.getDate(),
+    5
+  )
+  const shouldRefetch =
+    front.meta.path === '/' &&
+    lastFetchAttemptAtbyPath[front.meta.path] < dailyUpdateTime
```

Comment:
Would switch it around and pass to `isRefetching` as initial state:
```suggestion
  const now = new Date()
  const dailyUpdateTime = new Date(
    now.getFullYear(),
    now.getMonth(),
    now.getDate(),
    5
  )
  const shouldRefetch =
    front.meta.path === '/' &&
    lastFetchAttemptAtbyPath[front.meta.path] < dailyUpdateTime
  const [isRefetching, setIsRefetching] = useState(shouldRefetch)
```
This way we avoid flashing the old front before switching to the loader. `setIsRefetching(true)` can then be removed from the effect.

Reply:
done